### PR TITLE
Setting Animals DB

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Created a database named vet_clinic.
- Created a table named animals with columns
- Inserted the animal's data
- Wrote queries for the different quests.

**Screenshot with the results of queries**

2db
3db
4db
5db
6db
7db
8db

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name like '%mon'
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```
Comment (mckent05) [inline queries.sql:6]: - You have done great so far, but please attach a screenshot for result to this query, according to the project requirements 🙏

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+SELECT name From animals WHERE neutered=true AND (escape_attempts < 3);
+
+SELECT date_of_birth FROM animals WHERE (name='Agumon' Or name='Pikachu');
```
Comment (mckent05) [inline queries.sql:10]: - You have done great so far, but please attach a screenshot for the result to this query, according to the project requirements 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Bleu-Yves-Sopoude, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Schema was properly created ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job creating your animals table 👍 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Bleu-Yves-Sopoude, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**